harvester_e2e_tests/fixtures/images.py

```diff
@@ -11,7 +11,7 @@
                               "/Leap_15.3/images/openSUSE-Leap-15.3.x86_64-NoCloud.qcow2")
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="class")
 def opensuse_image(request, api_client):
```

Comment:
the fixture is shareable and will not be update in test session, so we can keep `session` scope.

Reply:
Changed.